ads/google/imaVideo.js

```diff
@@ -16,12 +16,15 @@
 
 import {CONSENT_POLICY_STATE} from '../../src/core/constants/consent-state';
 import {ImaPlayerData} from './ima-player-data';
-import {camelCaseToTitleCase, px, setStyle, setStyles} from '../../src/style';
+import {camelCaseToTitleCase, setStyle, toggle} from '../../src/style';
 import {getData} from '../../src/event-helper';
+import {htmlFor, htmlRefs, svgFor} from '../../src/static-template';
 import {isObject} from '../../src/core/types';
 import {loadScript} from '../../3p/3p';
 import {throttle} from '../../src/core/types/function';
 import {tryParseJson} from '../../src/json';
+// Source for this constant is css/amp-ima-video-iframe.css
+import {cssText} from '../../build/amp-ima-video-iframe.css';
```

Comment:
The new CSS files replicates the presentation as before. Rules to modify elements on certain states are now CSS selectors, and this Javascript file is simplified by merely annotating state and forgoing granular style control.